tsconfig.json

```diff
@@ -1,19 +1,13 @@
-{
-	"compilerOptions": {
-		"target": "es6",
-		"module": "commonjs",
-		"moduleResolution": "node",
-		"outDir": "out",
-		"lib": [ "es2016" ],
-		"sourceMap": true,
-		"skipLibCheck": true
-	},
-	"exclude": [
-		"node_modules",
-		"server"
-	],
-	"include": [
-		"src",
-		"test"
-	]
-}
\ No newline at end of file
+{
+	"compilerOptions": {
+		"target": "es6",
+		"module": "commonjs",
+		"moduleResolution": "node",
+		"outDir": "out",
+		"lib": ["es2016"],
+		"sourceMap": true,
+		"skipLibCheck": true
```

Comment:
In my opinion it is easier to read, search, and diff if this file is alphabetized.